Makefile

```diff
@@ -20,13 +20,13 @@ test-coverage: dep ## Run tests with coverage
 	@cat ./bin/cover.out
 
 build: dep ## Build the binary file
-	@go build -o bin/backend ./cmd/backend
+	@go build -o bin/backend .
  
 clean: ## Remove previous build
 	@rm -rf ./bin ./logs
  
 run: ## run simulation
-	@go run cmd/backend/main.go
+	@go run main.go $(ARGS)
```

Comment:
Where is `$(ARGS)` defined?

Reply:
This is pulling flags from the command line when you do `make run {some flags here}` and then appending on the `go run main.go {the flags here}`

Reply:
This is pulling flags from the command line when you do `make run {some flags here}` and then appending on the `go run main.go {the flags here}`

Reply:
for example if you want to run `go run main.go --config=/path/to/config` you would run `make run ARGS="--config=/path/to/config`